src/SUMMARY.md

```diff
@@ -32,7 +32,10 @@
   - [Field Replacements](templates/fields.md)
   - [Card Generation](templates/generation.md)
   - [Styling & HTML](templates/styling.md)
-  - [Checks and Errors](templates/errors.md)
+  - [Checks and Errors](templates/errors/errors.md)
+    - [Field Replacement Errors](templates/errors/replacement-errors.md)
+    - [Cloze Errors](templates/errors/cloze-error.md)
+    - [Empty Cards](templates/errors/empty-error.md)
```

Comment:
These filenames are in an errors/ folder, so including error in them is redundant. How about e.g.

templates/errors/field-replacements.md
templates/errors/cloze-deletions.md
templates/errors/empty-cards.md